desktop/lib/messagebundles/default.po

```diff
@@ -4269,6 +4269,25 @@ msgstr ""
 msgid "Enable Distributed Hash Table (DHT)"
 msgstr ""
 
+#: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/limegroup/gnutella/gui/options/panes/TorrentConnectionPaneItem.java:42
```

Comment:
@gubatron If you are talking about the yellow one - yes the layout is called view_rich_notification.xml